pkg/controllers/pushsecret/pushsecret_controller_test.go

```diff
@@ -38,7 +38,7 @@ import (
 
 var (
 	fakeProvider *fake.Client
-	timeout      = time.Second * 10
+	timeout      = time.Second * 20
```

Comment:
Seems like some of the tests take more that 10 seconds if we use the cached client 😢